src/shareglrm.jl

```diff
@@ -281,14 +374,14 @@ function fit!(glrm::GLRM; params::Params=Params(),ch::ConvergenceHistory=Converg
         if i>10 && (steps_in_a_row > 3 && ch.objective[end-1] - totalobj < tol) || alpha[1] <= params.min_stepsize
             break
         end
-        if verbose && i%10==0 
+        if verbose
             println("Iteration $i: objective value = $(ch.objective[end])") 
         end
     end
     t = time() - t
     update!(ch, t, ch.objective[end])
 
-    return glrm.X', glrm.Y, ch
+    return glrm.X.s, glrm.Y.s, ch, alpha[1]
```

Comment:
Type stability with `glrm.jl`. Shouldn't be a problem in the future when we parallelize everything.